spec/date.html

```diff
@@ -538,17 +556,17 @@ <h1>Properties of Temporal.Date Instances</h1>
     <h1>Abstract Operations for Temporal.Date Objects</h1>
 
     <emu-clause id="sec-temporal-createtemporaldate" aoid="CreateTemporalDate">
-      <h1>CreateTemporalDate ( _isoYear_, _isoMonth_, _isoDay_ [ , _newTarget_ ] )</h1>
+      <h1>CreateTemporalDate ( _isoYear_, _isoMonth_, _isoDay_, _calendar_ [ , _newTarget_ ] )</h1>
       <emu-alg>
         1. Perform ? RejectDate(_isoYear_, _isoMonth_, _isoDay_).
-        1. If ! DateTimeWithinLimits(_isoYear_, _isoMonth_, _isoDay_, 12, 0, 0, 0, 0, 0) is not ~in range~, then
-          1. Throw a *RangeError* exception.
+        1. If ! DateTimeWithinLimits(_isoYear_, _isoMonth_, _isoDay_, 12, 0, 0, 0, 0, 0) is not ~in range~, throw a *RangeError* exception.
+        1. If Type(_calendar_) is not Object, throw a *RangeError* exception.
```

Comment:
In the polyfill, it's allowed to pass a string for the calendar, which is run through Temporal.Calendar.from to get a calendar.

Reply:
I'll check the polyfill out and change this to match.

Reply:
The constructor behaviors match in the polyfill and spec. This is a lower level function than the constructor which doesn't (cannot) exist in the polyfill and I'd prefer we keep things this way.